diff --git a/julio.info b/julio.info
index 74d7b57..7e442a2 100644
--- a/julio.info
+++ b/julio.info
@@ -200,4 +200,23 @@ feature_sets[julio_clubs_announcements][enable][] = julio_clubs_announcements
 feature_sets[julio_clubs_announcements][enable][] = julio_clubs_announcements_ui
 feature_sets[julio_clubs_announcements][disable][] = julio_clubs_announcements
 feature_sets[julio_clubs_announcements][disable][] = julio_clubs_announcements_ui
-feature_sets[julio_clubs_announcements][uninstall][] = julio_clubs_announcements_ui
\ No newline at end of file
+feature_sets[julio_clubs_announcements][uninstall][] = julio_clubs_announcements_ui
+
+feature_sets[hpps_community][title] = Community Outreach
+feature_sets[hpps_community][icon] = "feature_set_icons/clipboard.png"
+feature_sets[hpps_community][description] = Add registration and supplies listing for community outreach.
+feature_sets[hpps_community][enable][] = hpps_community_event
+feature_sets[hpps_community][enable][] = hpps_global
+feature_sets[hpps_community][enable][] = hpps_registration
+feature_sets[hpps_community][enable][] = hpps_ongoing
+feature_sets[hpps_community][enable][] = hpps_supplies
+feature_sets[hpps_community][disable][] = hpps_community_event
+feature_sets[hpps_community][disable][] = hpps_global
+feature_sets[hpps_community][disable][] = hpps_registration
+feature_sets[hpps_community][disable][] = hpps_ongoing
+feature_sets[hpps_community][disable][] = hpps_supplies
+feature_sets[hpps_community][uninstall][] = hpps_community_event
+feature_sets[hpps_community][uninstall][] = hpps_global
+feature_sets[hpps_community][uninstall][] = hpps_registration
+feature_sets[hpps_community][uninstall][] = hpps_ongoing
+feature_sets[hpps_community][uninstall][] = hpps_supplies
